main.py

```diff
@@ -1,35 +1,77 @@
+''' Simple aiohttp login system with default user!:
+This system is built with aiohttp module.
+Required Python-3 or 3+
+'''
+
 from aiohttp import web
 
+# Customize the cookie name, value and cookie expire date
+COOKIE_NAME = "OldTamil"
+COOKIE_VALUE = 0
+COOKIE_LIFE_TIME = 24*60*60*30
 
-async def index(request):
-    cookie = request.cookies.get('default', None)
+# User credentials and here we could use more users in future.
+User ='Hippod'
+Password = 'admin'
+
+# Define the required host and port
+HOST = '127.0.0.1'
+PORT = 8080
+
+''' cookie_check function will check for valid cookie on all pages!
+Return True if there is cookie with requested name.
+'''
+def cookie_check(request):
+    cookie = request.cookies.get(COOKIE_NAME, None)
     if cookie:
+        if COOKIE_NAME == COOKIE_NAME:
+            return True
+        # Normal test for checking the COOKIE_NAME
+        else:
+            print("COOKIE_NAME has been modified!.")
+    # Return False when there is no cookie at all.
+    else:
+        return False
+
+# The main page of the website
+async def index(request):
+    Check = cookie_check(request)
+    if Check is True:
         page = open('site.html').read()
         return web.Response(text=page, content_type='text/html')
     else:
         return web.HTTPFound(location='/loginpage')
 
-
+# When the user is unknown or the user's cookie is expired, will redirect to login page
 async def login_page(request):
-    resp = open('login.html').read()
-    return web.Response(text=resp, content_type='text/html')
-
+    Check = cookie_check(request)
+    # check for cookies in the login_page and return to main page if there is valid cookie.
+    if Check is True:
+        return web.HTTPFound('/')
+    else:
+        resp = open('login.html').read()
+        return web.Response(text=resp, content_type='text/html')
 
+# Login required if the user is new or user's cookie expired and check the user credentials.
 async def login(request):
-    form = await request.post()
-    username = form.get('username')
-    if username == 'hippod':
-        password = form.get('password')
-        if password == 'admin':
-            response = web.HTTPFound(location='/')
-            response.set_cookie('default', value=1, max_age=30*86400, path='/')
-            return response
-    else:
-        resp = web.HTTPFound(location='/loginpage')
-        return resp
+        form = await request.post()
+        username = form.get('username')
+        if username == User:
```

Comment:
here, the problems comes into play, here `User` looks like a local variable! If you wrote it with UPPERCASE everybody knows instantly that you compare `username` with a global definition. Much more easy to follow the code.